### PR TITLE
New package: Crucial.StorageExecutive version 9.09.092023.03

### LINKED ISSUE #132973: [Package Request]: Crucial Storage Executive
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.crucial.de/support/storage-executive
Publisher: Crucial
Package Name: Crucial Storage Executive
Description: Crucial Storage Executive SSD / NVMe Management and Firmware
Package Version: ? 
Installer URL: https://www.crucial.de/content/dam/crucial/support/storage-executive-win-64.zip

### PR DESCRIPTION
**Pull request by Exorcism0666**
- Resolve: #132973

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132977)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by Exorcism0666** [comment]
image

**Comment by Exorcism0666** [comment]
> image

What do you think I should use for silent installation?

**Comment by Exorcism0666** [comment]
> image

@OfficialEsco what should i do ?

File: manifests/c/Crucial/StorageExecutive/9.09.092023.03/Crucial.StorageExecutive.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+PackageIdentifier: Crucial.StorageExecutive
+PackageVersion: 9.09.092023.03
+InstallerType: zip
+NestedInstallerType: exe
+NestedInstallerFiles:
+- RelativeFilePath: Storage-Executive-win-64.exe
+InstallerSwitches:
+  Silent: /S --force-run
+  SilentWithProgress: /S --force-run
```
Comment (OfficialEsco) [inline manifests/c/Crucial/StorageExecutive/9.09.092023.03/Crucial.StorageExecutive.installer.yaml:12]:
```suggestion
  Silent: --mode unattended --force-run
  SilentWithProgress: --mode unattended --unattendedmodeui minimal --force-run
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Azure Pipeline
> Installation failed with exit code -1978335226

Manual validation
image

Manual testing
image

File: manifests/c/Crucial/StorageExecutive/9.09.092023.03/Crucial.StorageExecutive.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+PackageIdentifier: Crucial.StorageExecutive
+PackageVersion: 9.09.092023.03
+InstallerType: zip
+NestedInstallerType: exe
+NestedInstallerFiles:
+- RelativeFilePath: Storage-Executive-win-64.exe
+InstallerSwitches:
+  Silent: --mode unattended --force-run
+  SilentWithProgress: --mode unattended --unattendedmodeui minimal --force-run
```
Comment (OfficialEsco) [inline manifests/c/Crucial/StorageExecutive/9.09.092023.03/Crucial.StorageExecutive.installer.yaml:12]:
```suggestion
  Silent: --mode unattended
  SilentWithProgress: --mode unattended --unattendedmodeui minimal
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Why did you add these? >.<
I didn't do my testing with that switch so i tough it was intended

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**